src/components/InputClass.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+class Input extends React.Component {
+    render(props, handleChange) {
+        return (
+            <div className={props.state.mainDivClassName}>
+            <label>{props.state.mainDivClassName== 'phone' && !props.state.filled? <p className="error">{props.textContent.textForError}</p>: null}
```

Comment:
it should be generic component please don't mention specific words like "phone", you should move such conditions out from this class